src/sentry/tagstore/types.py

```diff
@@ -72,21 +72,28 @@ def __init__(self, group_id, key, value, times_seen, first_seen, last_seen):
         self.last_seen = last_seen
 
 
-from sentry.api.serializers import Serializer, register
+from sentry.api.serializers import Serializer, register, serialize
 
 
+@register(GroupTagKey)
```

Comment:
Aside: I wonder if we ever return/serialize the actual model instances anymore or if we can delete all of these for sanity?

https://github.com/getsentry/sentry/blob/ae2ec4c15a7d84e73b0310696374c33a310eb7d1/src/sentry/tagstore/v2/models/grouptagkey.py#L99-L100

Reply:
Oh, I see you removed the v2 `TagKey` one, so yeah maybe remove these too.

Reply:
ok, to be clear you mean remove the serializer in `v2/models/grouptagkey.py` ?

Reply:
Yeah just any duplicate serializer since it seems like we aren't using them?